vignettes/combine-cut-interact.Rmd

```diff
@@ -0,0 +1,140 @@
+---
+title: "Combining Answers and Variables"
+description: "Vignette showing you how to take existing variables and recombine their answers or other variables."
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Adding Variables}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+[Previous: subtotals](subtotals.html)
+
+```{r, results='hide', echo=FALSE, message=FALSE}
+## Because the vignette tasks require communicating with a remote host,
+## we do all the work ahead of time and save a workspace, which we load here.
+## We'll then reference saved objects in that as if we had just retrieved them
+## from the server
+library(crunch)
+load("vignettes.RData")
+options(width=120)
+```
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+Many common data cleaning steps are grouping a number of categories or values together for easier analysis. For example, you might have a numeric variable that you want to transform from continuous to a set of bins. In Crunch, this is easy with the `cut()` command (just like base R!). Or, say you have a categorical variable with many different categories that you want to summarize by collapsing categories together. To do this as a separate variable, use `combine()`, if instead you want subtotals to be displayed alongside the original categories, you might want [subtotals](subtotals.html) instead. Finally, if you have multiple categorical variables `interactVariables()` can make interactions of many variables. Each of these will be discussed in this vignette.
+
+# Cutting a Numeric Variable into Categories
+Say we have a numeric type variable `age`, which is in years from 18-99, and we want to place each answer into one of a few categories: 18-29, 30-44, etc. We can use the `cut()` function to do just that and _cut_ the numeric variable into a new Categorical type variable. We designed this function to match the way that base R's `cut()` function works. 
+
+```{r, eval = FALSE}
+ds$age4 <- cut(ds$age, name="Age (4 categories)",
+  breaks=c(17,29,44,64,100), labels=c('18-29', '30-44', '45-64', '65+'))
+```
+```{r, eval = FALSE}
+categories(ds$age4)
+```
+```{r, echo = FALSE}
+cat(summary.age4.var, sep="\n")
+```
+
+And now we have a new Categorical type variable with the alias `age4`, the name "Age (4 categories)", and it's made of 4 categories.
+
+# Re-Combining Answer Choices
```

Comment:
When I read re-combining at first, I found it a bit awkward. I wonder if maybe just "Combining answer choices" would be better here?

Reply:
Yeah. Re-combining is not great. Let's switch